runtime/src/objc/cpp/ObjCExportClasses.mm

```diff
@@ -36,6 +36,9 @@ -(BOOL)_tryRetain;
 
 static void injectToRuntime();
 
+// Note: `KotlinBase`'s `toKotlin`, `retain` and `_tryRetain` methods will terminate if
```

Comment:
Not entirely true for `retain`: it will fail only if back ref's `refCount` is zero, which actually shouldn't happen.

Reply:
Same with `BackRefFromAssociatedObject::addRef`.